scipy/stats/_distn_infrastructure.py

```diff
@@ -2559,12 +2560,26 @@ def fit(self, data, *args, **kwds):
 
         >>> from scipy.stats import beta
         >>> a, b = 1., 2.
-        >>> x = beta.rvs(a, b, size=1000)
+        >>> x = beta.rvs(a, b, size=1000, random_state=80)
 
         Now we can fit all four parameters (``a``, ``b``, ``loc`` and
         ``scale``):
 
         >>> a1, b1, loc1, scale1 = beta.fit(x)
+        >>> a1, b1, loc1, scale1
+        (0.998314523745146, 2.0351090001496326, 0.0004988689720895926, 1.0059326261917274)
+
+        The fit can be done also using a custom optimizer:
+
+        >>> from scipy.optimize import minimize
+        >>> def custom_optimizer(func, x0, args=(), disp=0):
+        ...     res = minimize(func, x0, args, method="COBYLA")
```

Comment:
```suggestion
        ...     res = minimize(func, x0, args, method="COBYLA", disp=disp)
```

Reply:
thanks for reviewing, I will actually remove the disp argument from the docstring as it is anyways set to 0 when calling fit in stats/_distn_infrastructure.py
https://github.com/scipy/scipy/blob/1004bb958b43426cd6cfd5cdf5baafa4f138902c/scipy/stats/_distn_infrastructure.py#L2640C57-L2640C57

```python3
vals = optimizer(func, x0, args=(data,), disp=0)
obj = func(vals, data)
```